### ISSUE TITLE
CustomFilterTable error on Vaadin 7.5.1 

### ISSUE THREAD

**Issue by ofbizbrazil** [tepi/FilteringTable#91]
After migrating to Vaadin 7.5.1 appears on screen the following error:

```
Widgetset 'com.sbb.vaadin.gwt.VaadinWidgetset' does not contain implementation for com.sbb.fdn.vaadin.ui.table.data.CustomFilterTable. Check its component connector's @Connect mapping, widgetsets GWT module description file and re-compile your widgetset. In case you have downloaded a vaadin add-on package, you might want to refer to add-on instructions.
```

I'd appreciate any help!!

**Comment by gpedro** [comment]
Resolved in `0.9.13.v7`. Please close issue.
https://vaadin.com/directory#!addon/filteringtable

**Comment by juandavidgrisales** [comment]
you would put:

<servlet>
        .......
        <init-param>
                    <param-name>blablabla....</param-name>
                    <param-value>another.values....</param-value>
        </init-param>
        .......
        <init-param>
                    <param-name>widgetset</param-name>
                    <param-value>your.package.AppWidgetSet</param-value>
        </init-param>

</servlet>

and your.package.AppWidgetSet is the route for your file: AppWidgetSet.gwt.xml
dont forget replace "your.package" for your package jeje.
